shared/test/integration/helpers/pageRegion.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+var View = require('./view');
+
+function PageRegion(client, element) {
+	View.call(this, client);
+	this.waitForDisplay(element);
```

Comment:
In the Python version of PageRegions, we define an attribute called `root_element`. It helps to find something within this element, so you're sure you don't search outside of the region you defined.

One thing we don't do in Python, but could be great here: define a "findElement" method that make sure you use the root element. The reasoning behind putting it as a method of PageRegion is the following: if one day the API to find an element within another changes, you'll have to change on that place.